src/lib/components/ResultsGrid/ResultsGrid.js

```diff
@@ -32,11 +45,13 @@ ResultsGrid.propTypes = {
   loading: PropTypes.bool.isRequired,
   totalResults: PropTypes.number.isRequired,
   results: PropTypes.array.isRequired,
+  onRender: PropTypes.func,
 };
 
 ResultsGrid.defaultProps = {
   resultsPerRow: 3,
   overridableId: "",
+  onRender: undefined,
```

Comment:
documentation: could you maybe check if we have already documentation for this component on our [RSK docs](https://github.com/inveniosoftware/react-searchkit/tree/master/docs) and document also the new property?